recipes/flask-cache/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "flask-cache" %}
+{% set version = "0.13.1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://pypi.python.org/packages/91/c4/f71095437bd4b691c63f240e72a20c57e2c216085cbc271f79665885d3da/Flask-Cache-0.13.1.tar.gz
+  md5: ab82a9cd0844891ccdb54fbb93fd6c59
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask
+
+  run:
+    - python
+    - flask
+
+test:
+  imports:
+    - flask_cache
+
+about:
+  home: http://github.com/thadeusb/flask-cache
+  license: BSD
```

Comment:
It looks like the license is `BSD 3-Clause`. Can you confirm that and change it here?